__tests__/integration/test.bats

```diff
@@ -146,3 +151,22 @@ teardown() {
   assertFailedMkdocs build
   [[ "$output" == *"[mkdocs-monorepo] The /"*"/__tests__/integration/fixtures/error-include-path-no-docs-folder/project-a/docs path is not valid. Please update your 'nav' with a valid path."* ]]
 }
+
+@test "fails if !include paths included yml has an absolute url. Which results in a broken url" {
+  cd ${fixturesDir}/include-path-absolute-url
+  assertSuccessMkdocs build
+
+  assertFileContains './site/index.html' 'href="http://www.absoluteurl.nl"'
+  assertFileContains './site/index.html' 'href="https://www.absoluteurl.nl"'
+  # "{}/{}".format(alias, value) in parser.py results in 	'href="test/ftp://ftp.not-supported-absoluteurl.nl"'
+  # resulting test/index.html shows 						'href="test/ftp:/ftp.not-supported-absoluteurl.nl"'  though
+  # resulting ./index.html shows 							'href="test/ftp://ftp.not-supported-absoluteurl.nl"'
+  # looks like monorepo somehow changes the link. i.e. a slash is lost!
```

Comment:
Nice! Have you considered using some form of HTML selector (like [pup](https://github.com/ericchiang/pup)) for more accurate tests?

Reply:
I had not heard of pup. I checked the website it is quite powerful. I think that a grep will do for now but for more complex tests pup will do a better job.

Reply:
Sounds good :)